dbflute-engine/src/main/java/org/dbflute/logic/replaceschema/process/altercheck/DfAlterCheckProcess.java

```diff
@@ -1263,7 +1274,7 @@ protected void deleteFile(File file, String msg) {
 
     protected void copyFile(File src, File dest) {
         try {
-            FileUtils.getFileUtils().copyFile(src, dest);
+            Files.copy(src.toPath(), dest.toPath(), StandardCopyOption.COPY_ATTRIBUTES);
```

Comment:
I wrote the simple test to migrate FileUtils to Files.copy()
https://github.com/dbflute/dbflute-core/commit/e62a311ab097348eb53642d1d3ac6d9c1235a577#diff-3ce6d2e35f488be6c8ec4575f931b264R32

And I will write blackbox tests of AlterCheck.
So merging is after that.

Reply:
> And I will write blackbox tests of AlterCheck.
Now testing:
https://github.com/dbflute-test/dbflute-test-active-hangar/blob/master/src/test/java/org/docksidestage/hangar/dbflute/whitebox/task/WxDiffworldTest.java

It may take a long time to finish...

Reply:
I wrote.

But I want to test at Windows...